app/src/main/res/layout/activity_change_password.xml

```diff
@@ -1,79 +1,78 @@
 <?xml version="1.0" encoding="utf-8"?>
-<androidx.constraintlayout.widget.ConstraintLayout
-    xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:tools="http://schemas.android.com/tools"
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     tools:context=".ui.account.ChangePasswordActivity">
 
-  <EditText
-      android:id="@+id/userProfileOldPassword"
-      android:layout_width="wrap_content"
-      android:layout_height="wrap_content"
-      android:layout_marginStart="100dp"
-      android:layout_marginTop="220dp"
-      android:layout_marginEnd="101dp"
-      android:autofillHints=""
-      android:ems="10"
-      android:fontFamily="@font/roboto_light"
-      android:hint="@string/old_password"
-      android:inputType="textPassword"
-      android:minHeight="48dp"
-      android:textColor="@color/black"
-      app:layout_constraintEnd_toEndOf="parent"
-      app:layout_constraintHorizontal_bias="0.0"
-      app:layout_constraintStart_toStartOf="parent"
-      app:layout_constraintTop_toTopOf="parent" />
+    <EditText
+        android:id="@+id/userProfileOldPassword"
```

Comment:
I believe most of our `android:id`s use snake case instead of camel case, so maybe we update the items of this page (ex. `android:id="@+id/user_profile_old_password`. From what I can tell, they're still accessible in the bindings via a camel case reference.